WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/DashboardBlazeCampaignView.swift

```diff
@@ -0,0 +1,125 @@
+        titleLabel.numberOfLines = 2
+        titleLabel.font = WPStyleGuide.fontForTextStyle(.headline, fontWeight: .semibold)
+
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.layer.masksToBounds = true
+        imageView.layer.cornerRadius = 4
+
+        NSLayoutConstraint.activate([
+            imageView.widthAnchor.constraint(equalToConstant: 44),
+            imageView.heightAnchor.constraint(equalToConstant: 44),
+        ])
+
+        let headerView = UIStackView(arrangedSubviews: [titleLabel, imageView])
+        headerView.alignment = .top
+        headerView.spacing = 12
+
+        let contentView = UIStackView(arrangedSubviews: [
+            UIStackView(arrangedSubviews: [statusView, UIView()]), // Leading alignment
+            headerView,
+            statsView
+        ])
+        contentView.axis = .vertical
+        contentView.spacing = 8
+        contentView.setCustomSpacing(12, after: headerView)
+
+        addSubview(contentView)
+        contentView.translatesAutoresizingMaskIntoConstraints = false
+        pinSubviewToAllEdges(contentView)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func configure(with viewModel: DashboardBlazeCampaignViewModel, blog: Blog) {
+        statusView.configure(with: viewModel.status)
+
+        titleLabel.text = viewModel.title
+
+        imageLoader.prepareForReuse()
+        imageView.isHidden = viewModel.imageURL == nil
+        if let imageURL = viewModel.imageURL {
+            let host = MediaHost(with: blog, failure: { error in
+                WordPressAppDelegate.crashLogging?.logError(error)
+            })
+            imageLoader.loadImage(with: imageURL, from: host, preferredSize: Constants.imageSize)
+        }
+
+        statsView.isHidden = !viewModel.isShowingStats
+        if viewModel.isShowingStats {
+            statsView.arrangedSubviews.forEach { $0.removeFromSuperview() }
+            makeStatsViews(for: viewModel).forEach(statsView.addArrangedSubview)
+        }
+    }
+
+    private func makeStatsViews(for viewModel: DashboardBlazeCampaignViewModel) -> [UIView] {
+        let impressionsView = DashboardSingleStatView(title: Strings.impressions)
+        impressionsView.countString = "\(viewModel.impressions)"
+
+        let clicksView = DashboardSingleStatView(title: Strings.clicks)
+        clicksView.countString = "\(viewModel.clicks)"
+
+        return [impressionsView, clicksView]
+    }
+}
+
+private extension DashboardBlazeCampaignView {
+    enum Strings {
+        static let impressions = NSLocalizedString("my-sites.blazeCampaigns.impressions", value: "Impressions", comment: "Title for impressions stats view")
+        static let clicks = NSLocalizedString("my-sites.blazeCampaigns.clicks", value: "Clicks", comment: "Title for impressions stats view")
+    }
+
+    private struct Constants {
```

Comment:
[nit] enums as a non-initializable namespace
```suggestion
    enum Constants {
```

Reply:
Done